bfitzpat/fuseide
# FUSETOOLS-2458 - provide tests for bean global configuration handling of

node

I try to create some tests to avoid regression while working on the handling or not of GlobalBean Node

Currently 2 tests are failing

todo:
- verify that I wrote the assertions correctly
- verify that the conditions for tests are correct
- really fix the code
- do we need to add also for bean using ref with SAP installed and not installed? Do we handle this usecase currently?
